.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+#
+# For syntax help see:
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+*     @jiren @odeke-em @dazuma
```

Comment:
I think we should remove odeke-em as he no longer works on this project.

Reply:
Is he still working for Google?  Should I remove him from the org, and any teams he belongs to?

Reply:
I believe he is still working for Google but currently not on Spanner client libraries. So if it's ok, it might make sense to remove him and the we can add him back when we need to.